### PR TITLE
Update system reset warning message regarding deletion of volumes

### PR DESCRIPTION
**Pull request by bbalp**
#### Does this PR introduce a user-facing change?

```release-note
None
```

**TL;DR:** I almost lost an awful amount of work.

**Story:** Yesterday, I needed to use netavark, to have the DNS feature enabled on my network. As my podman install was pre-netavark, I had to use the system reset command to enable it. It warned me that almost everything was going to be removed. As I understoot it, everything except my volumes was going to be deleted ... I asked myself, do I trust that? Well, yes, it looks like an exhaustive list, and maybe because volumes are basic enough, it may not be needed to delete them. It will just reset the core features of podman I guess, right? Do I confirm reset then? I said Yes. You know what followed.

Fortunately, I ran a backup script for my volumes very recently. But it wasn't planed in the first place. I am VERY lucky not to have lost a week of work! ... and it was a dense and tricky kind of work on top of that.

**This is the motivation for this PR: Warning users that the volumes will get deleted too.** It is mentioned in the documentation, but not in the warning message shown to the user when running the command.

And folks, BACKUP, BACKUP, BACKUP!!!

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
Thanks a lot for fixing that!  The message is really misleading.

In order to pass CI you need to add the magical `[NO NEW TESTS NEEDED]` string to your commit message; your changes don't require a test.

**Review by Luap99 — APPROVED** [review]
LGTM

**Comment by rhatdan** [comment]
Thanks @bbalp
